src/main/java/tp/cap5buddy/todolist/Criterion.java

```diff
@@ -0,0 +1,7 @@
+package tp.cap5buddy.todolist;
+
```

Comment:
Perhaps javadoc comments could be added for this enumeration to further explain the role of this enum class?